galera/src/ist_proto.hpp

```diff
@@ -62,6 +74,8 @@ namespace galera
                 len_    (len    )
             { }
 
+            ~Message() { }
```

Comment:
I tried to make code build with -Weffc++ which required virtual destructor. However, this broke protocol due to dependency on Message class size, so I removed the virtual but left explicit destructor.